apps/website/src/pages/docs/radiotile.mdx

```diff
@@ -9,17 +9,28 @@ group: inputs
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='input-radiotilegroup' />
-
 <LiveExample src='RadioTile.main.tsx'>
   <AllExamples.RadioTileMainExample client:load />
 </LiveExample>
 
+## Usage
+
+A series of selectable tile components that behave like radio button. You may include icons to help make each tile more identifiable.
```

Comment:
I think the line "A series of selectable tile components that behave like radio button." should go in the `description` on line 3. Or add something else to the description on line 3 because it's missing one.

Reply:
This still needs to be updated.